### PR TITLE
chore: remove unnecessary words in the README

### PR DESCRIPTION
**Pull request by knanao**
I removed unnatural words in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ human and sometimes make mistakes. To prevent accidental breaking changes, you
 can also pin to a specific version:
 
 ```yaml
-- uses: 'google-github-actions/setup-gcloud@v0.1.1'
```
Comment (sethvargo) [inline README.md:217]: This was intentional. We won't be updating the README to point to the latest point release all the time.

**Review by sethvargo — CHANGES_REQUESTED** [review]
Aside from the main readme, these other changes look good.

File: README.md (hunk 1 above)
Comment (knanao) [reply README.md:217]: thank you for reviewing.
Oh, I see. Sorry for my misunderstanding.
Let me fix the title of PR.

**Review by sethvargo — APPROVED** [review]
Thanks!

**Pull request merged**